### PR TITLE
Adviser workflow notebook

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

Depends-On: https://github.com/thoth-station/common/pull/593

### CODE REVIEW

File: notebooks/workflows/adviser-inspection-workflow.ipynb [hunk 1]
```diff
@@ -0,0 +1,254 @@
+{
```
Comment (CermakM) [inline notebooks/workflows/adviser-inspection-workflow.ipynb:1]: <p>"stable" should be uppercase</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thoth-station/notebooks/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (CermakM) [inline notebooks/workflows/adviser-inspection-workflow.ipynb:1]: <p><code>json</code></p><p><code>&nbsp;&nbsp;"THOTH_ADVISER_LIBRARY_USAGE": {</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;"report": {},</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;"version": "string"</code></p><p><code>&nbsp;&nbsp;},</code></p><p> </p><p> </p><p>should be string, not an object</p><p> </p><p><code>json</code></p><p><code>"THOTH_ADVISER_LIBRARY_USAGE": "{ \"report\": {}, \"version\": \"string\" }",</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thoth-station/notebooks/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

**Review by CermakM — CHANGES_REQUESTED** [review]
See the comments

**Comment by CermakM** [comment]
@pacospace I think we should submit the notebook without outputs, WDYT?

**Comment by pacospace** [comment]
> @pacospace I think we should submit the notebook without outputs, WDYT?

Updated, thanks :)

**Review by CermakM — APPROVED** [review]

**Pull request merged**